docs/docs/cmd/entra/m365group/m365group-user-set.mdx

```diff
@@ -24,13 +24,25 @@ m365 aad teams user set
 
 ```md definition-list
 `-i, --groupId [groupId]`
-: The ID of the Microsoft 365 group for which to update user
+: The ID of the Microsoft 365 group. Specify only one of the following: `groupId`, `groupName`, `teamId`, or `teamName`.
+
+`--groupName [groupName]`
+: The display name of the Microsoft 365 group. Specify only one of the following: `groupId`, `groupName`, `teamId`, or `teamName`.
 
 `--teamId [teamId]`
-: The ID of the Microsoft Teams team for which to update user
+: The ID of the Teams team. Specify only one of the following: `groupId`, `groupName`, `teamId`, or `teamName`.
+
+`--teamName [teamName]`
+: The display name of the Microsoft Teams team. Specify only one of the following: `groupId`, `groupName`, `teamId`, or `teamName`.
+
+`-n, --userName [userName]`
+: User's UPN (User Principal Name), e.g. johndoe@example.com. This option is deprecated. Please use `ids` or `userNames` instead.
 
-`-n, --userName <userName>`
-: UPN of the user for whom to update the role (eg. johndoe@example.com)
+`--ids [ids]`
+: Microsoft Entra IDs of users. You can also pass a comma-separated list of IDs. Specify either `ids` or `userNames` but not both.
+
+`--userNames [userNames]`
+: The user principal names of users. You can also pass a comma-separated list of UPNs. Specify either `ids` or `userNames` but not both.
```

Comment:
For now, let's mention the three options that we can use, so userName, ids or userNames. This will be removed in v9.